.github/workflows/swift.yaml

```
@@ -8,7 +8,7 @@ on:
     tags:
       - 'v[0-9]+.[0-9]+.[0-9]+*'
     paths:
-      - './build-swift-macos.sh'
+      - './build-macos.sh'
```

actionlint (1.7.12)

[error] '.' and '..' are not allowed in glob path. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)